pkg/utils/client/client.go

```diff
@@ -34,6 +34,7 @@ func NewClientConfig() (*openapi.Config, error) {
 		return nil, errors.New("regionId must be set in the config file")
 	}
 
+	profile.Mode = aliyunconfig.AK
```

Comment:
we may need a better way to build the client,

Reply:
Why we need to specify this?

Reply:
https://github.com/Vacant2333/karpenter-provider-alicloud/blob/69c2d8802ede58479faf50f18ce368a8dc0126d3/vendor/github.com/aliyun/aliyun-cli/config/profile.go#L270 here,

Reply:
Maybe let's config the profile and set this field?